jq_service_parser.go

```diff
@@ -1,58 +1,21 @@
 package opslevel_jq_parser
 
-type JQServiceParser struct {
```

Comment:
Ultimately the output of JQ parser is OpsLevel API resources types inside a `ServiceRegistration`, so there is no need to really store this information here. 

Replaced with `RunWithConfig`.